examples/text_to_image/train_text_to_image.py

```diff
@@ -69,10 +69,10 @@ def save_model_card(
     args,
     repo_id: str,
     images: list = None,
-    repo_folder: str = None,
+    repo_folder: str = ".",
```

Comment:
I think this default value is perfectly okay, because for `repo_folder`, we use `args.output_dir`.

Reply:
Yes, but if `repo_folder` somehow takes its default value then isn't [this line](https://github.com/standardAI/diffusers/blob/68127d294705f2362d148f586e051125d6dd6d26/examples/text_to_image/train_text_to_image.py#L77) throws an error.